app/controllers/api/drops_controller.rb

```diff
@@ -2,6 +2,12 @@ class Api::DropsController < ApplicationController
   respond_to :json
 
   def index
-    @drops = Drop.all
+    if params[:latitude] && params[:longitude]
+      radius = params[:radius].presence || 10
```

Comment:
I would suggest creating a constant for the number `10`. 
I guess it would go in the drops model. So you could call it: `DEFAULT_RADIUS = 10`. Just so that we make sure we always use the same radius as default everywhere throughout the app & remain consistent. 

Let me know if you need help w/ this!